src/modules/project/createProject.js

```diff
@@ -0,0 +1,41 @@
+import displayController from '../displayController';
```

Comment:
## Fix the `createProject` functionality 
 - [ ] A user can not create a project as is required of this project. we are currently getting an error at the console.